### PR TITLE
Support Scala 2.13.0-M2

### PR DESCRIPTION
**Pull request by marcospereira**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -13,12 +13,12 @@ import com.typesafe.tools.mima.plugin.MimaKeys.{
 
 resolvers ++= DefaultOptions.resolvers(snapshot = true)
 
-val scala213Version = "2.13.0-M1"
+val scala213Version = "2.13.0-M2"
 
-val specsVersion = "3.9.1"
```
Comment (cchantep) [inline build.sbt]:
```scala
val specsBuild = Def.setting[Seq[ModuleID]] {
  val specsVersion = CrossVersion.partialVersion(scalaVersion.value) match {
    case Some((2, 10)) => "3.9.1"
    case _ => "4.0.1"
  }

  Seq("org.specs2" %% "specs2-core" % specsVersion)
}

// ...

libraryDependencies ++= specsBuild.value.map(_ % Test) // 'd prefer to have scope there
```

**Comment by gmethvin** [comment]
@marcospereira this has conflicts now; can you update. @cchantep's suggestion looks fine too.

**Comment by MasseGuillaume** [comment]
This PR fix https://github.com/playframework/play-json/issues/109

**Comment by marcospereira** [comment]
@gmethvin, the conflict was solved.

@cchantep please, review it again. :-)

**Comment by MasseGuillaume** [comment]
@marcospereira is it possible to cut a release for 2.13.0-M2? Thanks!

**Comment by marcospereira** [comment]
@MasseGuillaume just to let you know that play-json 2.6.9 was published just right now for 2.3.0-M3. :-)

**Pull request merged**